coalib/collecting/Collectors.py

```diff
@@ -82,7 +82,9 @@ def collect_files(file_paths, log_printer, ignored_file_paths=None,
         collected_files, file_globs_with_files = [], []
 
     _warn_if_unused_glob(log_printer, file_paths, file_globs_with_files,
-                         "No files matching '{}' were found.")
+                         "No files matching '{}' were found. "
+                         'If this rule is not required, you can remove it '
```

Comment:
I don't think I can. 
The double-quote to single-quote conversion was performed by coala (Quote-Bear?), when I was running it in the repository to verify my changes.
The format also matches other messages, that need inline single-quotes (i.e. the message warning about unmatched bears in [Collectors.py Line 179](https://github.com/coala/coala/blob/master/coalib/collecting/Collectors.py).

I will amend the autoformatting issues introduced by my editor as soon as I can